satpy/tests/reader_tests/test_eum_base.py

```diff
@@ -86,6 +86,14 @@ def test_fun(self):
             ('PlannedRepeatCycleEnd', time_cds_expanded)
         ])
 
+        test_mph = [
```

Comment:
Can you please split that in two test methods, for example `test_timestamps` and `test_mpef_header`?

Reply:
yes , will do

Reply:
Thanks!